### DISCUSSION TITLE
Custom line style

### DISCUSSION THREAD

**Discussion by gpsaliola** [Mapsui/Mapsui#1895] [General]
Is it possible create a line like this?

![example](https://user-images.githubusercontent.com/6804212/226163790-44db3d42-f18f-4dc6-8d50-8f90282c7946.jpg)

Thank you!

**Comment by pauldendulk** [comment]
No.

Although with a [CustomStyleRenderer](https://mapsui.com/documentation/custom-style-renders.html) you could do anything.

Ideas that might help:
- Draw a green line and use symbols for the black arrows (you would need an svg for that). Also the symbols need to be oriented in the right direction which may be hard. The number of symbols might depend on the zoom level which would make it hard. Perhaps you could use an ThemeStyle to return the symbols that you need.
- A LineString could be drawn with a Pen with a DashArray
- You could get black crossing lines by using a DashArray with.long intervals and  short dashes, and a high value for Pen.Width. But there would be no arrow.

Perhaps digging into the CustomStyleRenderer is the best option. Might be hard at first but you are in full control.

**Comment by gpsaliola** [comment]
Thanks for the tips.
I'm just afraid of performances.
